src/archivematicaCommon/lib/executeOrRunSubProcess.py

```diff
@@ -28,7 +28,8 @@
 import sys
 
 
-def launchSubProcess(command, stdIn="", printing=True, arguments=[], env_updates={}):
+def launchSubProcess(command, stdIn="", printing=True, arguments=[],
+                     env_updates={}, capture_output=False):
```

Comment:
The docstring says "defaults to `True`" but this is defaulting to `False`. Elsewhere, this defaults to `True` so I think it should be that way here as well.

Reply:
My original version was actually `capture_output=True` because that was the behaviour before this change, but @scollazo's patch https://github.com/JiscRDSS/archivematica/pull/42/commits/48a9d25421c4cbf6640d941612e6e45d9db47fcd (from upstream) changed this to `capture_output=False`. I think, going forward, that **not** capturing output by default is the correct way to go, so I think it should remain `capture_output=False`.

That said, I take your point about the docstring comment, so will update this to reflect the code (i.e. default is `False`).